spec/forms/idv/profile_form_spec.rb

```diff
@@ -59,7 +60,8 @@
   describe 'ssn uniqueness' do
     context 'when ssn is already taken by another profile' do
       it 'is invalid' do
-        create(:profile, ssn: '1234', user_id: 456)
+        diff_user = create(:user)
```

Comment:
This can be tested without using the DB. See this test as an example: https://github.com/18F/identity-idp/blob/master/spec/forms/update_user_email_form_spec.rb#L23-L33